src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -64,11 +64,27 @@ internal static TimeSeriesOperation Parse(BlittableJsonReaderObject input)
             return result;
         }
 
+        private const string TimeSeriesFormat = "TimeFormat";
+        private const string UnixTimeFormatMs = "UnixTimeFormatMs";
+        private const string UnixTimeFormatNs = "UnixTimeFormatNs";
+        private static readonly long _epochTicks = new DateTime(1970, 1, 1).Ticks;
+
+        private static long FromUnixMs(long unixMs)
+        {
+            return unixMs * 10_000 + _epochTicks;
+        }
+        private static long FromUnixNs(long unixNs)
+        {
+            return unixNs / 100 + _epochTicks;
+        }
+
         internal static TimeSeriesOperation ParseForBulkInsert(BlittableJsonReaderObject input)
         {
             if (input.TryGet(nameof(Name), out string name) == false || name == null)
                 ThrowMissingProperty(nameof(Name));
 
+            input.TryGet(TimeSeriesFormat, out string format);
```

Comment:
Hmm... I would prefer to convert this value to enum and if it exists parse it. If it cannot be parsed then we should throw.

Reply:
done